ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimDocumentMapperTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.ccd.config.CCDAdapterConfig;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaimDocument;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaimDocumentType;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.domain.models.ClaimDocument;
+import uk.gov.hmcts.cmc.domain.models.ClaimDocumentType;
+
+import java.net.URI;
+import java.time.LocalDateTime;
+
+import static org.junit.Assert.assertEquals;
+
+@SpringBootTest
+@ContextConfiguration(classes = CCDAdapterConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class ClaimDocumentMapperTest {
+
+    @Autowired
+    private ClaimDocumentMapper claimDocumentMapper;
+
+    private ClaimDocument claimDocument;
+
+    private static final String NAME = "foo";
```

Comment:
do we need these to be static?

Reply:
It doesn't "have" to be static. What would be the advantage of the alternative?

Reply:
static initializes variable at the start of the application. assume like lazy loading. So if not static it wont initialize it unless needed or used.